src/ModAttachr.ts

```diff
@@ -29,10 +30,15 @@ export class ModAttachr implements IModAttachr {
      * Transforms mod names to storage keys.
      */
     private readonly transformModName: ITransformModName;
-
+
+    /**
+    * class to hold modkeys instead of hardcoded strings
+    */
+    private readonly eventNames: EventNames;
+
     /**
      * Initializes a new instance of the ModAttachr class.
-     * 
```

Comment:
Ah, the trailing whitespace complaints... TSLint recently pushed an update to also check comments. They used to not.

I've pushed a quick commit fix to origin/master that removes the `no-trailing-whitespace` rule. Could you please pull it into this branch and undo these changes?